Sources/Classes/core/Display/MiniAppExternalWebViewController.swift

```diff
@@ -126,3 +127,19 @@ internal class MiniAppCloseNavigationController: UINavigationController {
         self.dismiss(animated: true)
     }
 }
+
+public struct MiniAppExternalWebView: UIViewControllerRepresentable {
+
+    public init() {
+        // init
+    }
+
+    public func makeUIViewController(context: Context) -> MiniAppExternalWebViewController {
+        return MiniAppExternalWebViewController()
+    }
+
+    public func updateUIViewController(_ uiViewController: MiniAppExternalWebViewController, context: Context) {
+        // update
```

Comment:
Any reason why we have this method ?

Reply:
it's part of the `UIViewRepresentable` and `UIViewControllerRepresentable` protocol conformance